### ISSUE TITLE
find typos, dead links

### ISSUE THREAD

**Issue by yash251** [dataverse-os/developer-docs#11]
### Current/Expected Behavior

Details:

- Typos:

Conduct a thorough review of the project documentation and codebase to identify and correct any typographical errors present.
Typos may include misspellings, grammatical errors, or inaccuracies that need correction for better clarity and professionalism.

- Dead Links:

Check all hyperlinks in the documentation and ensure they are functional.
Identify and report any dead links or broken references that may hinder the user's ability to access relevant information.

**Comment by manoranjith-shankar** [comment]
Hi @yash251 can i work on this issue?

**Comment by yash251** [comment]
> Hi @yash251 can I work on this issue?

yes definitely. If you come across any typos or deadlinks, feel free to create a PR

**Comment by kaksv** [comment]
Are multiple people allowed to work on the project @yash251 ?

**Comment by kaksv** [comment]
@yash251, sorry for asking but what is the easiest way to contribute to this repo, do I have to fork and clone only, I am getting errors while cloning.
